app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -183,6 +183,13 @@ def raw_metrics_for_counter(storage_conn, counter, start_time, end_time)
   end
 
   def get_counters(metrics_conn)
+    ems = target.ext_management_system
+
+    unless ems.supports_timeline?
```

Comment:
@bronaghs Upon further review it seems to cause test failures because of with_provider_connection which is called again later within the same method. I've moved it back for now.